service/src/test/java/tds/assessment/repositories/AssessmentQueryRepositoryImplIntegrationTests.java

```diff
@@ -92,9 +111,52 @@ public void shouldFindNonSegmentedAssessmentByKey() {
         assertThat(seg.getStartAbility()).isEqualTo(maybeAssessment.get().getStartAbility());
         assertThat(seg.getAssessmentKey()).isEqualTo(maybeAssessment.get().getKey());
         assertThat(seg.getSubject()).isEqualTo(maybeAssessment.get().getSubject());
+        assertThat(seg.getPosition()).isEqualTo(1);
+        assertThat(seg.getMinItems()).isEqualTo(4);
+        assertThat(seg.getMaxItems()).isEqualTo(4);
         assertThat(seg.getLanguages()).hasSize(1);
         assertThat(seg.getLanguages()).containsOnly(new Property("Language", "ENU"));
         assertThat(seg.getSelectionAlgorithm()).isEqualTo(maybeAssessment.get().getSelectionAlgorithm());
+
+        Form form1 = null;
+        Form form2 = null;
+        Form form3 = null;
+
+        for(Form form : seg.getForms()) {
+            switch (form.getKey()) {
+                case "187-510":
```

Comment:
Is there a reason we need to check against three different form records?  Does the `form2` differ from `form1` enough that it is exercising something different in the query?

If this check is just making sure that all three form records are mapped properly I don't think we need anything other than `form1`.  We should insert and check the bare minimum otherwise we end up with a lot of noise in the tests.